hack/pre-release.sh

```diff
@@ -187,9 +190,20 @@ switch_git_branch $base_branch $new_branch
 WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
 update_WMCO_version $stream $WMCO_VERSION
 update_OCP_version $stream $WMCO_VERSION 
+
+# check to see if the opeator-sdk version is correct 
+# if not, update it to the correct version 
```

Comment:
Please update the comment, commit and PR messages now that you are not automatically installing the SDK.